webflow-hacks/misc-hacks/20/20.ts

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const UL_SELECTOR = '[fs-hacks-element="unordered-list"]';
+  const originalList = document.querySelector(UL_SELECTOR);
+  const unorderedList = document.createElement('ul');
+  if (!originalList) return;
+
+  const listItems = originalList.querySelectorAll('li');
+
+  for (const item of listItems) {
+    const tildeCount = (item.innerText.match(/~/g) || []).length;
```

Comment:
I think we should only match `~` at the beginning of the text. Currently, you're checking for `~` in the entire strings, which means that this string:
> ~ Joe ate ~ 4 apples approximately.
Would count as 2 tildes, when in reality it should only be one.

Same appleis to the `cleanListItem` function, it should only clean the `~` at the beginning of the strings.

Reply:
In a list within a list, tildes can appear on the same line. So there are many cases where matching on the beginning of the line would render an invalid list but few cases where a semantic tilde would bring list formatting issue.